.github/workflows/tests.yml

```diff
@@ -52,7 +52,10 @@ jobs:
 
     - name: Build documentation
       run: |
-        meson compile doc -C build
+        rm -rf build/
+        meson setup -Dservice=false -Dcreate=false -Dnetwork=false -Dstreaming=false -Djson=disabled -Dgpt=disabled -Dtests=false -Dhtmldocs=true build
```

Comment:
I guess we should just leave the fields to their default and only set `-Dtests=false -htmldocs=true`.
We build the code anyway and build performance should not be an issue.